templates/default/apache2.conf.erb

```diff
@@ -67,6 +67,8 @@ Group <%= node['apache']['group'] %>
         Require all granted
 </Directory>
 
+DocumentRoot <%= node['apache']['docroot_dir'] %>
+
```

Comment:
I am 👎 on this.

We explicitly did not set the global `DocumentRoot` directive for security reasons, as it is best practice to use the `web_app` definition to create a `<VirtualHost>` which would include the `DocumentRoot` directive.

By including it globally, we would not have a way to turn it off and there is very little reason these days to enable it on a production apache httpd installation.

Reply:
OK, if this is the intention, then why not remove the <Directory> block on the same? In addition, I'm not sure of the purpose of the <Directory> on /usr/share/doc - seems like a loose end and confusing.

Reply:
I suppose this is a debatable point. If the main direction you want uses to go in is to create virtual sites, then maybe it is left as documentation (comments). For rhel/centos distros the default DocumentRoot is /etc/httpd/htdocs. So, this <Directory /var/www/html> can be very confusing. I went through that when learning the cookbook.

Maybe even better than what I did (replace with comments in httpd.conf) would be to also create an example Virtual Site configuration file, which is all comments, but serves as a handy reference to get people up and running. Put that file in place in sites-available and you would give the users something really helpful.